src/App.css

```diff
@@ -0,0 +1,69 @@
+.header {
```

Comment:
Let's split this file into more modular ones.

E.g. in the `Header` folder you will have a `.css` file with styles only for the header component. You will import those into the component itself.